docs/man/rpm-queryformat.7.scd

```diff
@@ -0,0 +1,290 @@
+inside a package, in a section called the _header_, and are identified by unique
+names called _tags_. The individual records are also referred to as _tag data_.
+
+You can use these tags in the query format to display the individual properties
+of a package. To obtain a list of all the tags supported by the version of *rpm*
+installed on your system, run the following command:
+
+```
+rpm --querytags
+```
+
+This will print a list of tags, one per line, for example:
+
+```
+BUILDHOST
+BUILDTIME
+DESCRIPTION
+EPOCH
+INSTALLTIME
+NAME
+RELEASE
+SIZE
+SUMMARY
+VERSION
+```
+
+Each tag is case-insensitive and also has an implicit variant with the *RPMTAG_*
+prefix, such as *RPMTAG_NAME*, which points to the same record. Both variants
+can be used in the query format interchangeably.
+
+## Formats
+
+Each tag is printed in a specific output _format_. When no format is specified,
+the default *:string* format is used which displays a bare-bones representation
+of the tag data. You can specify a different format for a tag by appending a
+colon followed by the format name to the tag name, for example:
+
+```
+%{INSTALLTIME:date}
+```
+
+This is useful when displaying a tag that's not stored in human-readable form,
+such as a size or date, or when you want to perform a formatting operation on
+the tag data before it's displayed, such as wrapping it in JSON.
+
+The following formats are available:
+
+*:armor*
+	Wrap a public key in ASCII armor.
+
+*:arraysize*
+	Display number of elements in array tags.
+
+*:base64*
+	Encode binary data using base64.
+
+*:date*
+	Use *strftime*(3) "%c" format.
+
+*:day*
+	Use *strftime*(3) "%a %b %d %Y" format.
+
+*:depflags*
+	Format dependency comparison operator.
+
+*:deptype*
+	Format dependency type.
+
+*:expand*
+	Perform macro expansion.
+
+*:fflags*
+	Format file flags.
+
+*:fstate*
+	Format file state.
+
+*:fstatus*
+	Format file verify status.
+
+*:hashalgo*
+	Display hash algorithm name.
+
+*:hex*
+	Format in hexadecimal.
+
+*:octal*
+	Format in octal.
+
+*:humaniec*
+	Human readable number (in IEC 80000). The suffix K = 1024, M =
+	1048576, ...
+
+*:humansi*
+	Human readable number (in SI). The suffix K = 1000, M = 1000000, ...
+
+*:json*
+	Wrap data in JSON.
+
+*:perms*
+	Format file permissions.
+
+*:pgpsig*
+	Display signature fingerprint and time.
+
+*:shescape*
+	Escape single quotes for use in a script.
+
+*:string*
+	Display string format. (default)
+
+*:tagname*
+	Display tag name.
+
+*:tagnum*
+	Display tag number.
+
+*:triggertype*
+	Display trigger suffix.
+
+*:vflags*
+	File verification flags.
+
+*:xml*
+	Wrap data in simple XML markup.
+
+## Field width
+
+Similarly to *printf*(3), you can specify the minimum width of a given tag when
+printed, by placing a number in between the *%* and *{* characters. Optionally,
+prefix the number with the *-* sign to left justify it.
+
+## Arrays
+
+*rpm* uses many parallel arrays internally. For example, file sizes and file
```

Comment:
This too obviously comes from the original text, but I think this needs a short paragraph about arrays and their syntax before diving into *parallel* arrays and their complexities.